quaddtype/numpy_quaddtype/__init__.pyi

```diff
@@ -10,9 +11,15 @@ from ._quaddtype_main import (
     set_num_threads,
 )
 
+class QuadBackend(enum.IntEnum):
+    """Backend type for QuadPrecision computations."""
```

Comment:
docstrings aren't needed in stubs (there's even a `flake8-pyi` rule that prohibits it)

```suggestion
```